src/main/resources/org/jenkinsci/plugins/oic/OicSecurityRealm/help-subjectType.html

```diff
@@ -0,0 +1,4 @@
+<div>
+	The Client secret used by jenkins to authenticate itself to the openid connect provider.
```

Comment:
not sure what happened here - but this has been swapped with the help-clientSecret and needs to be swapped back.